pom.xml

```diff
@@ -36,15 +36,22 @@
     </developers>
 
```

Comment:
I think we need to update version of the lib as well:

software.amazon.payloadoffloading
payloadoffloading-common
1.0.0. ==> to new version

Reply:
Would you want that to be 2.0.0?

Reply:
Yes this should be version 2.0.0, should include README.md change as well
`<version>2.0.0</version>`

Reply:
fixed